config.json

```diff
@@ -7,14 +7,30 @@
         },
         "idHash": {
             "algo": "sha256",
-            "hashLength": 24,
+            "bytes": 24,
             "iterations": 10000,
             "salt": ""
         },
         "initiateLifetime": {
             "hours": 1
         },
+        "loginDir": "/login/",
```

Comment:
You should split the login information into a new top level property because `account` is looking very crowded.